pandas/io/tests/parse_dates_tests.py

```diff
@@ -0,0 +1,471 @@
+"""
+Tests date parsing functionality for all of the
```

Comment:
TBH it seems easier to work in the way of making the individual test suites part of a `common` package and sub-class them into each of the main parser tests as is done now.  Otherwise, I fear there will be duplicate code IINM.  For example, for `parse_date_tests.py` needs to be run for each of three parsers.  But I can just throw all in as super-classes because they will clash.  So I would then have to make three different versions, one for each parser, which I don't think is preferable.